features/environment.py

```diff
@@ -666,6 +666,13 @@ def start(self, name, max_wait_limit=40, custom_config=None):
                                                       self._output_dir, custom_config)
         self._processes[name].start(max_wait_limit)
 
+    def start_with_expected_failure(self, name, max_wait_limit=40, custom_config=None):
+        try:
+            self.start(name, max_wait_limit, custom_config)
+            assert False, 'expected startup to fail'
+        except:
```

Comment:
_E722 do not use bare 'except'_

```suggestion
        except Exception:
```

Reply:
Done.